### PR TITLE
Adding task-lint app

### PR DESCRIPTION
**Pull request by boyney123**
A new app I built that checks task lists on a pull request and sets a Status check on the output. 

Goes red if there are tasks outstanding otherwise will be green.

##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.

### CODE REVIEW

File: _apps/task-lint.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Task-Lint
+description:  Status Check that checks comments in your pull requests and will fail if any of them have tasks outstanding.
```
Comment (hiimbex) [inline _apps/task-lint.md]: nit: can we change:
> Status Check that checks comments in your pull requests and will fail if any of them have tasks outstanding.

to
> Status Check that checks comments in your pull requests and will fail if there are any outstanding tasks.

I just think it's a little more readable.

File: _apps/task-lint.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+slug: task-lint
+screenshots:
+- https://avatars3.githubusercontent.com/in/15182?s=88&amp;v=4
```
Comment (hiimbex) [inline _apps/task-lint.md]: For this screenshot, I'd prefer if you'd use the other one from down below. This is where the screenshots appear for each app, see: https://probot.github.io/apps/stale/ for an example.

File: _apps/task-lint.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Usage
+
+Just install and use.
+
+![image](https://user-images.githubusercontent.com/3268013/43205901-5ac0c088-901c-11e8-8182-0ad72812674d.png)
+
+
+Task-lint will check your comments in your pull requests and will fail if any of them have tasks outstanding.
+                                                       |
```
Comment (hiimbex) [inline _apps/task-lint.md]: Can we nix this extra character?

**Comment by boyney123** [comment]
Thanks for the review @hiimbex 

> Hi @boyney123, thanks for the PR! I'm curious, does it check for tasks in the initial PR body as well? I think that would significantly increase the usefulness, and would prefer that feature to be implemented before adding this app to the website.

Great idea! Thanks, il work on this and fix the .md file, and submit for another review 👍

**Pull request closed without merging**